### PR TITLE
Random Battle: Update on AV

### PR DESCRIPTION
**Pull request by galbia**
Added some special cases

### CODE REVIEW

**Comment by Slayer95** [comment]
B-but @sirDonovan just told you to add <code>Random Battle</code> to the commit message :|

**Comment by RhythmsAreFromVibrations** [comment]
I'm helping galbia with that

**Comment by galbia** [comment]
ok yeah i did

**Comment by ascriptmaster** [comment]
Why Escavalier, Slowking, and Azumarill specifically?

**Comment by galbia** [comment]
They weren't influenced by the past formula and have viable av sets in
their tier
Il 25/apr/2015 00:08, "Kevin Lau" <notifications@github.com> ha scritto:

File: data/scripts.js [hunk 1]
```diff
@@ -2814,7 +2814,7 @@ exports.BattleScripts = {
 			item = 'DeepSeaTooth';
 		} else if (template.species === 'Spiritomb') {
 			item = 'Leftovers';
-		} else if (template.species === 'Scrafty' && counter['Status'] === 0) {
+		} else if ((template.species === 'Scrafty' || template.species === 'Escavalier' || template.species === 'Slowking' || template.species === 'Azumarill')  && counter['Status'] === 0) {
```
Comment (RhythmsAreFromVibrations) [inline data/scripts.js]: Didn't see this before, oops

'Azumarill') &nbsp;&&

there are two spaces between the ``)`` and the ``&&``'s

**Comment by ascriptmaster** [comment]
Escavalier would be approved for the other Assault Vest check... if it weren't for the fact that the Life Orb and Occa Berry checks come first. Perhaps we should consider moving the other Assault Vest check up? But if we do that, the question would be what other Pokemon could it affect?

**Comment by galbia** [comment]
Unrelates but I think that Life Orb shouldn't be an option for Pokemon slower than 55 base speed anyway as it is never used in competitive sets for Pokemon that slow

**Pull request closed without merging**